ui/views.py

```diff
@@ -4,6 +4,7 @@
 from django.conf import settings
 from django.contrib.staticfiles.templatetags.staticfiles import static
 from django.shortcuts import render
+from courses import models
```

Comment:
Can you change this to `from courses.models import Program`? It's nitpicky but it's a more specific import which is good for code cleanliness